lib/src/clojure_lsp/feature/add_missing_libspec.clj

```diff
@@ -27,9 +27,19 @@
       (z/sexpr zloc)
       (catch Exception _))))
 
+(defn get-keyword-ns [zloc] 
+  (when-let [kw-str (z/string zloc)]
+    (symbol (if (string/starts-with? kw-str "::")
+      (subs kw-str 2)
+      "nil"))))
```

Comment:
I don't think this is correct. This will produce a symbol named nil that isn't the same as `nil`.

Reply:
Yes you are right. Fixing that...